noxfile.py

```diff
@@ -0,0 +1,56 @@
+import nox
+
+source_files = ("httpx", "tests", "setup.py", "noxfile.py")
+
+
+@nox.session(reuse_venv=True)
+def lint(session):
+    session.install("autoflake", "black", "flake8", "isort")
+
+    session.run("autoflake", "--in-place", "--recursive", *source_files)
+    session.run(
+        "isort",
```

Comment:
Adding [`--project=httpx` here](https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox#diff-37409d3afed5d01396a659e1b20bdf08R14) _should_ fix the first-party-is-third-party import swap, but for some reason it runs one way on my machine and the other way in a Docker container I built to test. I did end up getting it to be consistent by adding [`seed-isort-config`](https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox#diff-37409d3afed5d01396a659e1b20bdf08R11) and specifying the first party `httpx` in an [`.isort.cfg` file](https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox#diff-170b3556059bb1e212db57e7c98d02b6).

The following branch should be the correct sorting including `httpx`: https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox

The [Dockerfile I used](https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox#diff-3254677a7917c6c01f55212f86c57fbf) for testing is based off the one suggested in the nox docs, but only includes python 3.6, 3.7, and 3.8.

Reply:
Awesome! I'll use this as I'm sure it'll be a problem in the future if it's happening now.

Reply:
Cool! I should have also noted that I needed to move `httpx` in the generated config to [known_first_party](https://github.com/sethmlarson/http3/compare/nox...StephenBrown2:nox#diff-170b3556059bb1e212db57e7c98d02b6R2), _then_ isort ran correctly.

Reply:
It looks like isort ran correctly for me while using ``--project=httpx`? Should I add it to isort.cfg anyways?

Reply:
I would, because then the config won't conflict with the command line arg. It should be overridden anyways, but it wasn't working for me in this state for some reason.

Reply:
LGTM!

I'll let the real reviewers review now. :-p

Reply:
Thank you for your review! :heart: